msgpack/include/adaptor.hpp

```diff
@@ -80,32 +77,25 @@ namespace msgpack
                 {
 
                     if(v.valueType() == ValueType::Nil)
-                    {
                         o.pack_nil();
-                    }
                     else if(v.valueType() == ValueType::True)
-                    {
                         o.pack_true();
-                    }
                     else if(v.valueType() == ValueType::False)
-                    {
                         o.pack_false();
-                    }
                     else if(v.valueType() == ValueType::Number)
-                    {
                         o.pack_double(static_cast<Value>(v).number());
-                    }
                     else if(v.valueType() == ValueType::String)
                     {
-                        std::string str { static_cast<Value>(v).stringRef().toString() };
+                        std::string str = static_cast<Value>(v).string().toString();
                         o.pack_str(str.size());
                         o.pack_str_body(str.c_str(), str.size());
```

Comment:
You don't need to cast to std::string using toString() here, .size() and .c_str() are available on Ark::String, it will also be more efficient this way